extra/utils/Validation.js

```diff
@@ -1,3 +1,12 @@
+feature/39519
+import {user} from '../constant'
+import {validateEmail} from './helpers'
+
+
+let a =[];
+let b =[];
+export default function validateUsers(users){
+
 let users = [
```

Comment:
What is this duplicate code. Kindly go through this file

Reply:
I gone through this file and resolve the duplicate code as well as file .